services/app-api/handlers/formTemplates/write.ts

```diff
@@ -15,24 +17,46 @@ export const writeFormTemplate = handler(async (event, _context) => {
       body: UNAUTHORIZED_MESSAGE,
     };
   }
-  const body = JSON.parse(event!.body!);
-  if (!body?.formTemplateId) {
+  const unvalidatedPayload = JSON.parse(event!.body!);
+  if (!unvalidatedPayload?.formTemplateId) {
     throw new Error(NO_KEY_ERROR_MESSAGE);
-  } else if (!body?.formTemplate) {
+  } else if (!unvalidatedPayload?.formTemplate) {
     throw new Error(MISSING_DATA_ERROR_MESSAGE);
   } else {
-    const params = {
-      TableName: process.env.FORM_TEMPLATE_TABLE_NAME!,
-      Item: {
-        formTemplateId: body.formTemplateId,
-        createdAt: Date.now(),
-        lastAltered: Date.now(),
-        lastAlteredBy: event?.headers["cognito-identity-id"],
-        formTemplate: body.formTemplate,
-        ...body,
-      },
-    };
-    await dynamoDb.put(params);
-    return { status: StatusCodes.SUCCESS, body: params };
+    const validationSchema = yup.object().shape({
```

Comment:
Wondering if we want to turn these schemas into "objects/models" of sorts (to use the Java term). Meaning we'd store them elsewhere. Either in a types-esque file or in separate files per schema

Reply:
Not a bad idea.